nipype/interfaces/tests/test_nilearn.py

```diff
@@ -0,0 +1,199 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+import unittest
+import os
+import tempfile
+import shutil
+
+import numpy as np
+
+from ...testing import (assert_equal, utils, assert_almost_equal, raises,
+                        skipif)
+from .. import nilearn as iface
+
+no_nilearn = True
```

Comment:
I was using the recent dvars testing file as a model (https://github.com/nipy/nipype/blob/master/nipype/algorithms/tests/test_confounds.py), which used nitime. I'm guessing that needs to be fixed too? Since this and the dvars interfaces have already been merged, I opened a separate issue for this: https://github.com/nipy/nipype/issues/1661